### PR TITLE
adding notifications template for generator

### PR DESCRIPTION
**Pull request by larrymou9**
Adding notifications template

### CODE REVIEW

**Comment by larrymou9** [comment]
hi @jhutar made the following additions to trigger generator for Notifications to send kafka messages. Let me know if you have any questions

File: opl/generators/notifications_ingress.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import jinja2
+import jinja2.meta
+
+import opl.gen
+import opl.generators.packages
+import opl.generators.generic
+import generic
+
+
+class EgressHostsGenerator(generic.GenericGenerator):
```
Comment (jhutar) [inline opl/generators/notifications_ingress.py]: Please call the generator something else. Like `NotificationsMessagesGenerator` or so.

File: opl/generators/notifications_ingress.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+        super().__init__(count=count, template=template, dump_message=False)
```
Comment (jhutar) [inline opl/generators/notifications_ingress.py:8]: Given `GenericGenerator` seems to have same interface as your `__init__`, I think you can remove `__init__` at all.

**Comment by jhutar** [comment]
Also please fix linter complaints.

File: opl/generators/notifications_ingress.py (hunk 1 above)
Comment (larrymou9) [reply opl/generators/notifications_ingress.py]: Hi @jhutar did this change :)

File: opl/generators/notifications_ingress.py (hunk 2 above)
Comment (larrymou9) [reply opl/generators/notifications_ingress.py:8]: For this one, I tried to do it as you suggested, but I was getting a runtime error

Comment (larrymou9) [reply opl/generators/notifications_ingress.py:8]: I have simplify it though

File: opl/generators/notifications_ingres_template.json.j2 [hunk 3]
```diff
@@ -0,0 +1,18 @@
+i{
```
Comment (jhutar) [inline opl/generators/notifications_ingres_template.json.j2]: Should the `i` be here?

File: opl/generators/notifications_ingress.py (hunk 2 above)
Comment (jhutar) [reply opl/generators/notifications_ingress.py:8]: What was the error? Anyway happy to merge it like this once the other comment is resolved.

File: opl/generators/notifications_ingress.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+import opl.generators.generic
+
+
+class NotificationsMessagesGenerator(opl.generators.generic.GenericGenerator):
+    def __init__(self, count=1, template="notifications_ingres_template.json.j2"):
+        super().__init__(count=count, template=template, dump_message=False)
+
+    def _data(self):
+        account = self._get_account()
```
Comment (jhutar) [inline opl/generators/notifications_ingress.py:11]: Given you only use that variable once, might be nicer to just put `self._get_account()` below. But again, this it totally not a blocker.

**Review by jhutar — APPROVED** [review]
LGTM

**Pull request merged**